msu/hooks/ui/screens/tactical/modules/turn_sequence_bar/turn_sequence_bar.nut

```diff
@@ -28,7 +28,29 @@
 			}
 		}
 
+		local table = {
+			__JSHandle = this.m.JSHandle;
+			function asyncCall( _funcName, ... )
+			{
+				if (_funcName == "updateCostsPreview")
+					return;
+
+				vargv.insert(0, _funcName);
+				vargv.insert(0, this);
+				this.__JSHandle.asyncCall.acall(vargv);
+			}
+		}
+		table.setdelegate({
+			function _get( _key )
+			{
+				if (_key in this.__JSHandle) return this.__JSHandle[_key];
+				throw null;
+			}
+		});
```

Comment:
Probably define it in .m once and then refer to it instead of redefining it every time.

Reply:
Done